index.js

```diff
@@ -0,0 +1,35 @@
+// npm module entry
```

Comment:
Add semicolons ... I know this is a religious thing ... but we are currently using them everywhere :) Please;

Reply:
Yes, I tried but apparently missed some by habit 😅 I will add them soon. The project does not seem to have any JS linter configured. Personally, I like to use (Javascript Standard Style](https://standardjs.com/) and its linter but its rules are quite different than what has been followed in the openmoji code base.